src/component/common/Table/SortableTableHeader/SortableTableHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import { VFC } from 'react';
+import { TableHead, TableRow } from '@mui/material';
+import { HeaderGroup } from 'react-table';
+import { useStyles } from './SortableTableHeader.styles';
+import { CellSortable } from './CellSortable/CellSortable';
+
+interface ISortableTableHeaderProps {
+    headerGroups: HeaderGroup<object>[];
+}
+
+export const SortableTableHeader: VFC<ISortableTableHeaderProps> = ({
+    headerGroups,
+}) => {
+    const { classes: styles } = useStyles();
+    return (
+        <TableHead>
+            {headerGroups.map(headerGroup => (
+                <TableRow
+                    {...headerGroup.getHeaderGroupProps()}
+                    className={styles.tableHeader}
+                >
+                    {headerGroup.headers.map(column => {
+                        const content = column.render('Header');
+
+                        return (
+                            <CellSortable
+                                {...column.getHeaderProps(
+                                    column.getSortByToggleProps()
+                                )}
+                                ariaTitle={
+                                    typeof content === 'string'
+                                        ? content
+                                        : undefined
+                                }
```

Comment:
It's a limitation of react-table. It will be set as long as we define `Header: string` for every column. Lib author is open to PRs about a11y https://github.com/TanStack/react-table/issues/130